### ISSUE TITLE
Problem with installing veil frame work

### ISSUE THREAD

**Issue by kallemAnvesh** [Veil-Framework/Veil#241]
## Veil version
Veil 3.1.X

## OS Used - all info (architecture, linux flavor, etc)
4.14.0-kali3-amd64 #1 SMP Debian 4.14.12-2kali1 x86_64 GNU/Linux

## How did you install Veil? (Apt, Clone from Github, etc.)
Clone

## Did you run the setup script?
yes i Did. The setup.sh file was in CONFIG folder

## Pastebin link to error you are encountering (include console actions you took prior to error)
command before error: - **/Downloads/Veil#  ./config/setup.sh --force --silent**
Error: -  **[ERROR] There was issues installing the following:

 
Failed with installing dependencies (1): 0
Failed with installing dependencies (6): 0
Failed with installing dependencies (5): 0
Failed with installing wine (2): 0
Failed to run (wine) Python pip... Exit code: 193
Failed to run (wine) Python pip future... Exit code: 193
Failed to run (wine) Python pip pefile... Exit code: 193
Failed to install (Wine) Ruby.exe... Exit code: 193
Failed to run (wine) Ruby OCRA.. Exit code: 49
Failed to run (wine) AutoIT.. Exit code: 193

 [I] If you have any errors running Veil, run: './Veil.py --setup' and select the nuke the wine folder option**

## Any additional info you want to tell me

I have tried ./Veil.py --setup command but the error after using this command is 

**[!] ERROR #2-3: Can't find the WINE profile for Ruby v1.8.7 (/var/lib/veil/wine//drive_c/Ruby187/bin/ruby.exe).   Run: /root/Downloads/Veil/config/setup.sh --force --silent**

**Comment by ChrisTruncer** [comment]
Can you check the repos that you are using? I just verified Veil to be running and installing fine on the latest version of Kali that was release a day or two ago.

**Comment by kallemAnvesh** [comment]
Yes I have checked and here are the repositories

**# deb cdrom:[Debian GNU/Linux 2018.1 _Kali-rolling_ - Official Snapshot amd64 LIVE/INSTALL Binary 20180126-21:23]/ kali-last-snapshot contrib main non-free

#deb cdrom:[Debian GNU/Linux 2018.1 _Kali-rolling_ - Official Snapshot amd64 LIVE/INSTALL Binary 20180126-21:23]/ kali-last-snapshot contrib main non-free**

**Comment by ChrisTruncer** [comment]
Unfortunately, it must be something isolated with your system. I am running the latest Kali and I was able to successfully install Veil. If you can provide me steps to reproduce this, I will be happy to look into it. However at the moment I can not reproduce the error.

**Comment by usama7628674** [comment]
@kallemAnvesh Edit your sources.list and remove everything then append the following lines and do `apt update` 
```
deb https://http.kali.org/kali kali-rolling main non-free contrib
```

**Comment by ChrisTruncer** [comment]
Did this work for you @kallemAnvesh?

**Comment by aPatinthehat** [comment]
@kallemAnvesh are you running headless by any chance? If so, use `-X` when you ssh to the box.

**Comment by ChrisTruncer** [comment]
Closing due to no response

**Comment by AchrafElhadi** [comment]
i fixed the probleme 
you should go to terminal and write : leafpad /etc/apt/sources.list .
and you remove all and write this 2 sentence :

deb http://http.kali.org/kali kali-rolling main non-free contrib
deb-src http://http.kali.org/kali kali-rolling main non-free contrib

and save it

after that , you go to terminal and write apt-get update 
wait untill finish and write apt-get upgrade

after that write : ./setup.sh --silent --force

and you will have no probleme inshaallah

if you still have the probleme try this command apt-get install wine32 and apt-get install wine and
 apt-get install winetricks

**Comment by pranesh92** [comment]
@ChrisTruncer @achrafesodhso 

I have tried all the above mentioned lines like changing the source list and apt update , and then started the setup with slient and force still i see sames issues. I am unable to install wine wine32 and winetricks also as standalones. 

Please help me  
![error](https://user-images.githubusercontent.com/64418408/85743158-d0757480-b6d1-11ea-91b0-ec54add490af.png)

**Comment by tamermh** [comment]
> 
> 
> i fixed the probleme
> you should go to terminal and write : leafpad /etc/apt/sources.list .
> and you remove all and write this 2 sentence :
> 
> deb http://http.kali.org/kali kali-rolling main non-free contrib
> deb-src http://http.kali.org/kali kali-rolling main non-free contrib
> 
> and save it
> 
> after that , you go to terminal and write apt-get update
> wait untill finish and write apt-get upgrade
> 
> after that write : ./setup.sh --silent --force
> 
> and you will have no probleme inshaallah
> 
> if you still have the probleme try this command apt-get install wine32 and apt-get install wine and
> apt-get install winetricks

This workaround worked for me and fixed the problem.
Thanks

**Comment by schizophrene246** [comment]
I have the same problem. Please I would be glad if someone can help

**Comment by highrider0602** [comment]
> > i fixed the probleme
> > you should go to terminal and write : leafpad /etc/apt/sources.list .
> > and you remove all and write this 2 sentence :
> > deb http://http.kali.org/kali kali-rolling main non-free contrib
> > deb-src http://http.kali.org/kali kali-rolling main non-free contrib
> > and save it
> > after that , you go to terminal and write apt-get update
> > wait untill finish and write apt-get upgrade
> > after that write : ./setup.sh --silent --force
> > and you will have no probleme inshaallah
> > if you still have the probleme try this command apt-get install wine32 and apt-get install wine and
> > apt-get install winetricks
> 
> This workaround worked for me and fixed the problem.
> Thanks

tried this and all other suggestions and nothing worked. still having same issue. anyone have any other ideas?

**Comment by Deniasjr** [comment]
> i fixed the probleme
> you should go to terminal and write : leafpad /etc/apt/sources.list .
> and you remove all and write this 2 sentence :
> 
> deb http://http.kali.org/kali kali-rolling main non-free contrib
> deb-src http://http.kali.org/kali kali-rolling main non-free contrib
> 
> and save it
> 
> after that , you go to terminal and write apt-get update
> wait untill finish and write apt-get upgrade
> 
> after that write : ./setup.sh --silent --force
> 
> and you will have no probleme inshaallah
> 
> if you still have the probleme try this command apt-get install wine32 and apt-get install wine and
> apt-get install winetricks

tried this, but nothing changed..

**Comment by iyappan04** [comment]
update your kali linux os 2020.3 to 2020.4

**Comment by hasnainmshabbir** [comment]
> 
> 
> update your kali linux os 2020.3 to 2020.4

i have 2021.3 but Veil is not getting installed can you please help me ?

**Comment by anandhanshine** [comment]
> I have the problem Failed with installing dependencies (6): 0

**Comment by Pakrykaas** [comment]
Anyone has any idea of what can be?


![Screenshot from 2023-10-20 17-03-12](https://github.com/Veil-Framework/Veil/assets/118173333/09edba4e-7f6e-4219-aa7a-40b297babe0c)

Tried installing from Github, after running: 

 ./config/setup.sh --force 
 
 The same issue as above was displayed. 
 
 When running: 
 
 /usr/share/veil/config/setup.sh --force --silent
 
The same output is printed

Im on Debian 12, above the specified Debian 8 in the error.

Since nothing seemed to work I added the kali repo along the GPG key. The installation went smoothly, but when running Veil the same problem was faced

Any tips on what can be done?
